package.json

```diff
@@ -128,7 +128,7 @@
     "parse-dashboard": "./bin/parse-dashboard"
   },
   "engines": {
-    "node": ">=8.9"
+    "node": ">=12.22.6"
```

Comment:
Can you change to `>=12`?

Reply:
As far as I investigated, MFA feature now depends on  `"inquirer": "8.1.2"` as seen on my screenshot from the pull request MFA.

<img width="1512" alt="Capture d’écran 2021-09-08 à 18 54 04" src="https://user-images.githubusercontent.com/644360/132551965-5dcebffb-50c9-468f-a6f5-1349e84149a2.png">

The new version of inquirer require ora package on version 5.3.0
The ora package need node 10 at least. (https://github.com/sindresorhus/ora/blob/476935f318868265303d148992fc268639a0d573/package.json)

So my i was wrong bumping to node 12. 10 is the new minimum (sorry for that).

Reply:
I can confirm to you node 10 is ok, i've tried 8, 10 and 12 with docker build on my laptop.
8 = X fails due to ora package.
10 => ok
12 => ok

Reply:
`node-qrcode` also requires "node": ">=10.13.0", and even worse, `otpauth` does not specific any min. node version.